toolkit/data-sources/db-sync/src/block/mod.rs

```diff
@@ -20,22 +21,41 @@ use std::{
 #[cfg(test)]
 mod tests;
 
+/// Db-Sync data source that queries Cardano block information
+///
+/// This data source does not implement any data source interface used by one of the
+/// Partner Chain toolkit's features, but is used internally by other data sources
+/// that require access to Cardano block data
 #[allow(clippy::too_many_arguments)]
 #[derive(new)]
 pub struct BlockDataSourceImpl {
+	/// Postgres connection poool
 	pool: PgPool,
+	/// Cardano security parameter
+	///
+	/// This parameter controls how many confirmations are required by the Cardano
+	/// node to consider a block to be stable. This is a network-wide parameter.
 	security_parameter: u32,
-	/// `security parameter / active slot coefficient` - minimal age of a block to be considered valid stable in relation to some given timestamp
+	/// Minimal age of a block to be considered valid stable in relation to some given timestamp.
+	/// Must be equal to `security parameter / active slot coefficient`.
 	min_slot_boundary_as_seconds: TimeDelta,
 	/// a characteristic of Ouroboros Praos and is equal to `3 * security parameter / active slot coefficient`
 	max_slot_boundary_as_seconds: TimeDelta,
+	/// Cardano main chain epoch configuration
 	mainchain_epoch_config: MainchainEpochConfig,
+	/// Additional offset applied when selecting the latest stable Cardano block
+	///
+	/// This parameter should be 0 by default and should only be increased to 1 for
+	/// Partner Chain nodes that experience frequent Db-Sync or Cardano node lag.
```

Comment:
This is misunderstanding. If given node experience frequent lags in blocks observation, setting this to 1 will not help it at all. 
This setting only prevents your block to be rejected by the nodes that are lagging. In other words, one can set it to help the network, but not help himself.